eternalcore-core/build.gradle.kts

```diff
@@ -7,9 +7,10 @@ plugins {
 
 dependencies {
     // modules
-    implementation(project(":eternalcore-api"))
-    implementation(project(":eternalcore-paper"))
-    implementation(project(":eternalcore-docs-api"))
+    implementation(project(mapOf("path" to ":eternalcore-api")))
+    implementation(project(mapOf("path" to ":eternalcore-paper")))
+    implementation(project(mapOf("path" to ":eternalcore-docs-api")))
```

Comment:
Nowy standard zmniejsza czytelność. Pytanie czy to nie nadinżynieria? Dasz linka do tego